### DISCUSSION TITLE
Use more Discussion instead of Issues

### DISCUSSION THREAD

**Discussion by manno** [epinio/epinio#814] [General]
Our issue list is growing. A lot of issues are not solvable (see https://en.wikipedia.org/wiki/SMART_criteria).

We also have an issue with the size of issues. Sometimes issues are marked as epics, but it's unclear how to deal with them. Split them and close the original epic, create multiple PRs, etc.
Some issues are only added as stubs, so they appear for the next milestone.

See for example: 

* #361 
* #806 

However, we did enable Github Discussions and we can convert issues to discussions. :)
We can then discuss what we want and create issues from that.

The only problem I see is that discussions cannot be part of milestones. So we don't have the epics/stub issues show up in the milestone view.

Limitations:
<img width="434" alt="converting issue to discussion" src="https://user-images.githubusercontent.com/393653/133759823-29fcc72a-4885-4220-95a6-a838ec014339.png">

I created this discussion, so we can try it out! So what do you think?

**Comment by kkaempf** [comment]
It's certainly worth a try. If it doesn't work for us, we can close discussions again.

**Reply by manno** [reply]
We can even convert them back into issues.
